src/main/java/org/assertj/core/error/ShouldContain.java

```diff
@@ -63,8 +64,8 @@ public static ErrorMessageFactory directoryShouldContain(Path actual, List<Strin
     return new ShouldContain(actual, directoryContent, filterDescription);
   }
 
-  private ShouldContain(Object actual, Object expected, Object notFound, ComparisonStrategy comparisonStrategy) {
-    super("%nExpecting:%n <%s>%nto contain:%n <%s>%nbut could not find:%n <%s>%n%s", actual, expected, notFound,
+  private ShouldContain(Object actual, Object expected, Object notFound, ComparisonStrategy comparisonStrategy, String[] name) {
+    super("%nExpecting" + name[0] + ":%n <%s>%nto contain:%n <%s>%nbut could not find the following "+name[1]+":%n <%s>%n%s", actual, expected, notFound,
```

Comment:
use `%s` instead of string concatenation

Reply:
this comment applies on other files

Reply:
I find use %s here and pass a String in this super() will show the double quotes, which can be a little strange to read

Reply:
yeah, fair enough let's go back to your initial implementation, just add a comment to say it will avoid quoted strings